integration-tests/tests/di_diun.rs

```diff
@@ -14,7 +14,12 @@ async fn test_diun() -> Result<()> {
     let mfg_server = ctx
         .start_test_server(
             Binary::ManufacturingServer,
-            |cfg| Ok(cfg.prepare_config_file(None, |_| Ok(()))?),
+            |cfg| {
+                cfg.prepare_config_file(Some("rendezvous-info.yml"), |ctx| {
+                    Ok(ctx.insert("rendezvous_port", &1337))
```

Comment:
this is just a dummy port - it's not needed for the test anyway and no reason to bring up the rendezvous either - let me know if anybody feels otherwise